cmd/pctl/upgrade.go

```diff
@@ -36,12 +41,24 @@ func upgradeCmd() *cli.Command {
 }
 
 func upgrade(c *cli.Context) error {
-	if c.Args().Len() != 2 {
-		return fmt.Errorf("please provid the path to the profile and version to upgrade to, e.g. pctl upgrade my-profile/ v0.1.1")
+	var (
+		profilePath, profileVersion string
+	)
+
+	latest := c.Bool("latest")
+	if latest {
+		if c.Args().Len() != 1 {
+			return fmt.Errorf("please provid the path to the profile to upgrade to, e.g. pctl upgrade my-profile/")
+		}
+		profilePath = c.Args().Slice()[0]
+	} else {
+		if c.Args().Len() != 2 {
+			return fmt.Errorf("please provid the path to the profile and version to upgrade to, e.g. pctl upgrade my-profile/ v0.1.1")
+		}
+		profilePath = c.Args().Slice()[0]
+		profileVersion = c.Args().Slice()[1]
```

Comment:
what about if `latest` was a valid value, rather than a flag :thinking: ?

Reply:
I don't really like that. If you see the previous push, I would have to then mock about with senver to compare the generated tags and values. Adds complexity. Also this is more explicit in my mind and more easy to discover as an option. While the <version> option doesn't make me think about , oh let's try writing `latest` as version to see what happens.